### DISCUSSION TITLE
Can't upgrade after disabling StrimziPodSets

### DISCUSSION THREAD

**Discussion by kingnarmer** [strimzi/strimzi-kafka-operator#7124] [General]
I ran helm upgrade to 0.30.0 then kafka/zookeeper got coverted to podsets. Then rolled back to 0.29.0 which brought back kafka/zookeeper statefulsets as expected. 

Then tried to upgrade to 0.30.0 disabling UseStrimziPodSets. The operator got upgraded to 0.30.0 , however cruise-control/entity-operator/kafka/zookeeper still @0.29.0.

I don't see any errors on operator logs. 

Appreciate any help

Thanks

 
Upgrade command 

 helm upgrade strimzi strimzi/strimzi-kafka-operator --version=0.30.0 -n strimzi-operator --set featureGates="-UseStrimziPodSets"

**Comment by scholzj** [comment]
Duplicate of https://cloud-native.slack.com/archives/CMH3Q3SNP/p1659027843615829 I guess? Issue was caused by incorrect configuration of the watched namespace.
